app/scripts/modules/alicloud/src/common/footer.directive.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.alicloud.footer.directive', []).directive('alicloudFooter', function() {
+  return {
+    restrict: 'E',
+    templateUrl: require('./footer.directive.html'),
+    scope: {},
+    bindToController: {
+      action: '&',
+      isValid: '&',
+      cancel: '&',
+      account: '=?',
+      verification: '=?',
+    },
+    controllerAs: 'vm',
+    controller: angular.noop,
+  };
+});
```

Comment:
HI @Jammy-Louie Thanks for your review. But, I am sorry to tell you that we have finished most of Deck works for a new Alibaba Cloud module using React and Angular. Can it is merged at first and then we will use angular to do the next developing works?

Reply:
> Can it is merged at first and then we will use angular to do the next developing works?

@xiaozhu36 just to clarify, did you mean you will use React to do future work? As @Jammy-Louie said, we try to only accept new code in React. Thanks!

Reply:
HI @maggieneterval Yes, I will use React to do future work. But my finished work still contains Angular. At the beginning, about 4 months ago, @jkschneider told me using React or Angular is ok, so my current most works use have react and angular. And also I can not found any guide says the deck only accept new code in React. 
I think my finished work can be accepted at first, because the code can not affect the current codes, on the another side, this acceptance is important for us and our customers.
After that, I will use react to develop new codes, and I can also consider to change our regular to react. But now, I think accept it and make it run is in first priority.

Reply:
Hey @xiaozhu36 that makes sense, and we should definitely do a better job documenting current conventions in Deck's [wiki](https://github.com/spinnaker/deck/wiki/Conventions). We would absolutely love to have you attend the [Spinnaker UI SIG](https://www.spinnaker.io/community/governance/sigs/#existing-sigs), where we often discuss best practices in Deck.

It would be great to get a thumbs-up from someone on the Netflix team before we merge this, @erikmunson or @alanmquach would you have time to take a quick look? :pray: